### ISSUE TITLE
npm dependencies won't install on linux

### ISSUE THREAD

**Issue by peterhebert** [SU-SWS/stanford_basic#240]
**Describe the bug**
cannot install the npm dependencies on linux. Get errors:
```
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  undefined
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64
```

**To Reproduce**

1. go to theme root in terminal
2. run `nvm install 14.8.0`
3. run `nvm use`
4. run `npm install`
5. See error

**Expected behavior**
Npm dependencies install, able to run build scripts.

**Screenshots**

**System information:**
 - OS: Linux pop-os 5.11.0-7620-generic #21~1626191760~20.04~55de9c3-Ubuntu SMP Wed Jul 21 20:31:55 UTC  x86_64 x86_64 x86_64 GNU/Linux

**Comment by sherakama** [comment]
@peterhebert,

Most of our developers are on OSX but we have had our windows counterparts remove that dependency to get things to install. We really should move that out of the dependencies and into optional in package.json.

**Comment by peterhebert** [comment]
Yes, that's what I ended up doing, removing that dependency. Good idea to
move it into optional...

Peter Hebert
***@***.***
https://peterhebert.com/
514-424-5682
